docs/installation/normal-installation.rst

```diff
@@ -232,189 +228,27 @@ This will run a default, example model, completing after reaching year 2050. The
     ...
     -- 2023-08-02 09:11:50 - muse.sectors.sector - INFO
     Running gas for year 2050
-    
+
     -- 2023-08-02 09:11:50 - muse.interactions - INFO
     Net new_to_retro of 1 interactions interacting via transfer
-    
+
     -- 2023-08-02 09:11:50 - muse.hooks - INFO
     Computing initial_asset_transform: default
-    
+
     -- 2023-08-02 09:11:50 - muse.hooks - INFO
     Computing initial_asset_transform: clean
-    
+
     -- 2023-08-02 09:11:50 - muse.demand_share - INFO
     Computing demand_share: default
-    
+
     -- 2023-08-02 09:11:51 - muse.production - INFO
     Computing production: max
-    
+
     -- 2023-08-02 09:11:51 - muse.production - INFO
     Computing production: max
-    
+
     -- 2023-08-02 09:11:51 - muse.production - INFO
     Computing production: share
-    
+
     -- 2023-08-02 09:11:51 - muse.mca - INFO
     Finish simulation year 2050!
-
-
-Alternative installation instructions
--------------------------------------
-
-If you don't want to use ``pyenv`` or ``pipx``, or if you are having trouble with those tools, there are a couple of alternatives.
-
-Installing Anaconda Python
-~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-We have chosen ``pyenv`` above because it is extremely lightweight and unobtrusive with your operative system. However, you might want to consider a more fully fledged Python distribution like Anaconda, specially if your work involved non-python packages or a lot of data science and machine learning tools.
-
```

Comment:
Maybe this paragraph on pyenv could be joined with lines 264. Then, there is a clear separation with the section on Anaconda

Reply:
Done!